bigbluebutton-html5/imports/ui/components/audio/captions/select/component.jsx

```diff
@@ -64,6 +68,19 @@ const Select = ({
   locale,
   voices,
 }) => {
+  if (!enabled || SpeechService.useFixedLocale()) {
+    return (
+      <div
+        style={{
+          fontSize: '.75rem',
+          padding: '1rem 0',
+        }}
+      >
+        {`*${intl.formatMessage(intlMessages.captionsNotEnabled)}`}
```

Comment:
Could we say nothing here, please?

If the feature is available (i.e. enabled on the server/meeting), and (browser does not support it) display that the browser does not support it;
If the feature is available (i.e. enabled on the server/meeting), and (browser supports it) display the current dropdown (no changes)
If the feature is not available (i.e. disabled on the server/meeting), and (browser supports it), don't display anything about it
If the feature is not available (i.e. disabled on the server/meeting), and (browser does not support it), don't display anything about it